### PR TITLE
Move area row

### PR DESCRIPTION
**Pull request by itkg-alavieille**
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/525
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1435

### CODE REVIEW

File: ApiBundle/Transformer/AreaFlexTransformer.php [hunk 1]
```diff
@@ -66,6 +72,50 @@ public function transformFromTemplateFlex(AreaFlexInterface $area, TemplateFlexI
...
+        $subAreaFacade = $facade->getAreas();
+        $newOrderSubAreas = array();
+        /** @var AreaFlexInterface $subArea */
+        foreach ($source->getAreas() as $subArea) {
+            $order = $this->getAreaOrderInChildren($subArea->getAreaId(), $subAreaFacade);
+            $newOrderSubAreas[$order] = $subArea;
+        }
+        ksort($newOrderSubAreas);
+        var_dump(count($newOrderSubAreas));
+        var_dump(new ArrayCollection($newOrderSubAreas));
```
Comment (Thiblef) [inline ApiBundle/Transformer/AreaFlexTransformer.php]: remove dumps

**Comment by itkg-alavieille** [comment]
@Thiblef pr updated

**Comment by Thiblef** [comment]
:+1:

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**